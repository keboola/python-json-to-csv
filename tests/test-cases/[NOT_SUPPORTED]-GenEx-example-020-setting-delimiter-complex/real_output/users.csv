id,name,primary.address_street,primary.address_city,primary.address_tags,primary.address_state,secondary.address_street,secondary.address_city,secondary.address_tags,secondary.address_state
123,John Doe,Elm Street,New York,users.primary.address_29ec9775ee6eb8e29eba998ad8fe3d87,,Cemetery Ridge,New York,users.secondary.address_81fcff2276712198e20de1367dada1a2,
234,Jane Doe, Blossom Avenue,,users.primary.address_8669aa264189d3fa1a97944f80164d02,U.K.,1313 Webfoot Walk,Duckburg,,Calisota
