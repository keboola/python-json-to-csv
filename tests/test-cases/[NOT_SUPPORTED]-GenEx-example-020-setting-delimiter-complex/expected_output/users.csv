"id","name","primary_address_street","primary_address_city","primary_address_tags","primary_address_state","secondary_address_street","secondary_address_city","secondary_address_tags","secondary_address_state"
"123","John Doe","Elm Street","New York","[]","","Cemetery Ridge","New York","[""work"",""usaddress""]",""
"234","Jane Doe"," Blossom Avenue","","[""home""]","U.K.","1313 Webfoot Walk","Duckburg","","Calisota"
